<ui version="4.0" >
 <class>TestLEDLightingClass</class>
 <widget class="QWidget" name="TestLEDLightingClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>TestLEDLighting</string>
  </property>
  <widget class="QPushButton" name="pbClose" >
   <property name="geometry" >
    <rect>
     <x>470</x>
     <y>500</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text" >
    <string>Close</string>
   </property>
  </widget>
  <widget class="QDockWidget" name="dwTestInput" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>321</width>
     <height>541</height>
    </rect>
   </property>
   <property name="features" >
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <widget class="QWidget" name="dockWidgetContents" >
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <widget class="QFrame" name="fmTestInput" >
       <property name="frameShape" >
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QFrame" name="fmBar" >
        <property name="geometry" >
         <rect>
          <x>110</x>
          <y>30</y>
          <width>51</width>
          <height>461</height>
         </rect>
        </property>
        <property name="autoFillBackground" >
         <bool>true</bool>
        </property>
        <property name="frameShape" >
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QTableWidget" name="twBar" >
         <property name="geometry" >
          <rect>
           <x>10</x>
           <y>10</y>
           <width>25</width>
           <height>265</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="leBar" >
         <property name="geometry" >
          <rect>
           <x>10</x>
           <y>280</y>
           <width>25</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="pbOneOn" >
        <property name="geometry" >
         <rect>
          <x>230</x>
          <y>260</y>
          <width>60</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text" >
         <string>One On</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pbAllOff" >
        <property name="geometry" >
         <rect>
          <x>10</x>
          <y>150</y>
          <width>91</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text" >
         <string>All Off</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lbBar" >
        <property name="geometry" >
         <rect>
          <x>110</x>
          <y>10</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text" >
         <string>Bar</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="pbAllOn" >
        <property name="geometry" >
         <rect>
          <x>10</x>
          <y>70</y>
          <width>91</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text" >
         <string>All On</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pbDetails" >
        <property name="geometry" >
         <rect>
          <x>10</x>
          <y>450</y>
          <width>91</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text" >
         <string>Details</string>
        </property>
       </widget>
       <widget class="QFrame" name="fmBrightness" >
        <property name="geometry" >
         <rect>
          <x>230</x>
          <y>30</y>
          <width>51</width>
          <height>211</height>
         </rect>
        </property>
        <property name="autoFillBackground" >
         <bool>true</bool>
        </property>
        <property name="frameShape" >
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QTableWidget" name="twBrightness" >
         <property name="geometry" >
          <rect>
           <x>10</x>
           <y>10</y>
           <width>25</width>
           <height>150</height>
          </rect>
         </property>
        </widget>
        <widget class="QSpinBox" name="sbBrightness" >
         <property name="geometry" >
          <rect>
           <x>4</x>
           <y>170</y>
           <width>42</width>
           <height>22</height>
          </rect>
         </property>
         <property name="maximum" >
          <number>40</number>
         </property>
        </widget>
       </widget>
       <widget class="QCheckBox" name="chkMultiple" >
        <property name="geometry" >
         <rect>
          <x>230</x>
          <y>460</y>
          <width>61</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text" >
         <string>Multiple</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lbLED" >
        <property name="geometry" >
         <rect>
          <x>170</x>
          <y>10</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text" >
         <string>LED</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="lbBrightness" >
        <property name="geometry" >
         <rect>
          <x>230</x>
          <y>10</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text" >
         <string>Bright</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QFrame" name="fmLED" >
        <property name="geometry" >
         <rect>
          <x>170</x>
          <y>30</y>
          <width>51</width>
          <height>461</height>
         </rect>
        </property>
        <property name="autoFillBackground" >
         <bool>true</bool>
        </property>
        <property name="frameShape" >
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QTableWidget" name="twLED" >
         <property name="geometry" >
          <rect>
           <x>10</x>
           <y>10</y>
           <width>25</width>
           <height>420</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="leLED" >
         <property name="geometry" >
          <rect>
           <x>10</x>
           <y>435</y>
           <width>25</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="pbNextOn" >
        <property name="geometry" >
         <rect>
          <x>230</x>
          <y>330</y>
          <width>60</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text" >
         <string>Next On</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dwContinueTest" >
   <property name="geometry" >
    <rect>
     <x>330</x>
     <y>10</y>
     <width>241</width>
     <height>271</height>
    </rect>
   </property>
   <property name="features" >
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <widget class="QWidget" name="dockWidgetContents_2" >
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <widget class="QFrame" name="fmContinueTest" >
       <property name="frameShape" >
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLineEdit" name="leSwitchTime" >
        <property name="geometry" >
         <rect>
          <x>70</x>
          <y>40</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text" >
         <string>Switch Time</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QSpinBox" name="sbSwitchTime" >
        <property name="geometry" >
         <rect>
          <x>70</x>
          <y>60</y>
          <width>71</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maximum" >
         <number>99999</number>
        </property>
       </widget>
       <widget class="QPushButton" name="pbStartPosClear" >
        <property name="geometry" >
         <rect>
          <x>10</x>
          <y>110</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text" >
         <string>Start position clear</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pbStart" >
        <property name="geometry" >
         <rect>
          <x>10</x>
          <y>170</y>
          <width>201</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text" >
         <string>Start</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QProgressBar" name="pgbLight" >
   <property name="geometry" >
    <rect>
     <x>340</x>
     <y>290</y>
     <width>221</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value" >
    <number>0</number>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
